flyingpigeon/processes/wps_plot_spatial_analog.py

```diff
@@ -7,6 +7,7 @@
 from pywps import LiteralInput
 from pywps import Process
 from pywps.app.Common import Metadata
+from pywps.ext_autodoc import MetadataUrl
```

Comment:
we might extend `Metadata` instead of using `MetadataUrl`:
https://github.com/geopython/pywps/pull/542

Reply:
Would you be able to create a new pywps release containing the change?

Note I've locked down the commit hash for pywps in this PR so using an official pywps release is not a blocking issue for now.  We can always make a subsequent PR to switch to an official pywps release.

Reply:
I can make a release when the `Metadata` is merged.